src/schemas/types.json

```diff
@@ -160,6 +160,50 @@
             "default": 0,
             "minimum": 0,
             "maximum": 9999
+        },
+        "Dimensions": {
+            "type": "object",
+            "properties": {
+                "width": {
+                    "type": "integer",
+                    "minimum": 1
+                },
+                "height": {
+                    "type": "integer",
+                    "minimum": 1
+                }
+            },
+            "required": [ "width", "height" ]
+        },
+        "Image": {
+            "type": "object",
+            "properties": {
+                "uri": {
+                    "description": "URI for the image. May be a relative path (e.g. ./foo/image.png) or absolute (e.g. https://foo.com/bar.png) depending on usage.",
+                    "type": "string"
+                },
+                "aspectRatio": {
+                    "description": "The aspect ratio of the image",
+                    "type": "string",
+                    "enum": [
+                        "1x1", "4x3", "16x9", "2x3", "4x5"
```

Comment:
Seems odd this is an enumeration, we should support any aspect ratio right?

Reply:
I can change this to just use a regex to confirm that it is of the form "<int>:<int>". That work?